pkg/capabilities/capabilities.go

```diff
@@ -40,6 +40,14 @@ func Initialize(c Capabilities) {
 	})
 }
 
+// Setup the capability set.  It wraps Initialize for improving usibility.
+func Setup(allowPrivileged bool, hostNetworkSources []string) {
```

Comment:
We didn't have a better concept at the time and the standard was evolving. With the global capabilities object this doesn't make much sense outside of the Kubelet.